apps/admin/src/app/workspace/workspace-layout-navigation-items.component.scss

```
@@ -0,0 +1,147 @@
+.nav-item {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 0;
+
+  width: var(--workspace-nav-item-size);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)